packages/create/src/index.ts

```diff
@@ -319,6 +319,41 @@ async function init() {
       ),
       "utf8",
     ),
+    fs.promises.writeFile(
+      path.join(root, "tsconfig.cjs-test.json"),
```

Comment:
just added this so when we generate the hatchify-app locally with `npm run pree2e` it's a bit more convenient to paste this in, though iirc the only actual change is `module: "CommonJS` and removing moduleResolution. it probably makes more sense to remove this and update json that is being written to `tsconfig.json` to just have 
```
// esm
module...
moduleResolution...

// cjs
// module...
// moduleResolution...

...rest of config
```
or something

Reply:
the generated app should always use `bundler` moduleResolution since it's using Vite?

Reply:
as for `type: module` vs `type: cjs` i don't know enough about the impact that has on typescript to comment? i guess it just determines if they export cjs or esm in the browser bundle?

i'd say lets opt for esm unless it has other impacts i'm not following